addons/mrp_subcontracting_dropshipping/models/stock_warehouse.py

```diff
@@ -15,6 +15,54 @@ class StockWarehouse(models.Model):
         'stock.rule', 'Subcontracting-Dropshipping MTS Rule'
     )
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        # if new warehouse has resupply enabled, enable global route
+        if any(['subcontracting_dropshipping_to_resupply' in vals for vals in vals_list]):
```

Comment:
if any([vals.get('subcontracting_dropshipping_to_resupply', False) for vals in vals_list]):